src/services/beers.js

```diff
@@ -42,6 +41,6 @@ export async function fetchBeers(param) {
       })
     )
   } catch (error) {
-    console.log(error)
```

Comment:
Leave error logs until you don't have some error handling approach

Reply:
Done!